fcrepo-kernel/src/main/java/org/fcrepo/kernel/observer/FedoraEvent.java

```diff
@@ -59,6 +65,7 @@
      */
     public FedoraEvent(final Event e) {
         checkArgument(e != null, "null cannot support a FedoraEvent!");
+        eventID = pidMinter.mintPid();
```

Comment:
We could add a `PidMinter` parameter to the `FedoraEvent` constructor and inject the pidMinter instance into the `AllNodeEventsOneEvent`, spring-managed bean.
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/observer/eventmappings/AllNodeEventsOneEvent.java#L50